pkgs/development/libraries/libfive/default.nix

```diff
@@ -26,7 +27,7 @@ stdenv.mkDerivation {
   };
 
   nativeBuildInputs = [ wrapQtAppsHook cmake ninja pkg-config ];
-  buildInputs = [ eigen zlib libpng boost guile qtbase ]
+  buildInputs = [ eigen zlib libpng boost guile python3 qtbase ]
```

Comment:
Only add python if built with `toPythonModule`, this can be condition on a input like `, enablePython ?? false`, which the version in `pkgs/top-level/python-packages.nix` sets to `true`

Reply:
This likely could be done, but it seems strange to exclude python to me.